core/blockchain.go

```diff
@@ -1377,8 +1376,15 @@ func (bc *BlockChain) InsertReceiptChain(blockChain types.Blocks, receiptChain [
 
 		// Write all chain data to ancients.
 		td := bc.GetTd(first.Hash(), first.NumberU64())
+		// TODO(GalaIO): if you just back fill blob, you need check blob ancient and reset it
```

Comment:
Oh, I will opt for it to be simpler. Because freezerdb is composed of closely arranged 2GB files, continuous and uninterrupted elements need to be passed in, and the order needs to be guaranteed for each insert, so the blob table needs to be additionally initialized here.